datafusion/core/src/physical_plan/common.rs

```diff
@@ -393,7 +393,10 @@ mod tests {
             ]),
         };
 
-        assert_eq!(result, expected);
+        // Prevent test flakiness due to undefined / changing implementation details
```

Comment:
This makes sense (and is something we see in IOx as well -- that the exact size of the parquet file changes from release to release). I think at least part of the discrepancy is that the version and gitsha of the arrow-rs library are encoded in the parquet metadata